src/fee-accounts/guards/accounts.guard.ts

```diff
@@ -11,6 +11,8 @@ export class AccountsGuard implements CanActivate {
     constructor(private store: Store<fromStore.OrganisationState>) { }
 
     canActivate(): Observable<boolean> {
+        // purposefully not allowing the user to hit Fee and Accounts tab
```

Comment:
I would place this as a block level comment above canActivate(). As I believe we shouldn't have // comments within our functions. The rest of the code is fine, as I know we just need to hide the fee and account tab. An interesting document is https://google.github.io/styleguide/jsguide.html#jsdoc which documents how Google uses Jsdocs.

Reply:
Its merged sir, but point noted. Thanks.